CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_AUTOMOC ON)
 
-find_package(MaliitPlugins 0.99.1 REQUIRED)
+find_package(MaliitPlugins 2.1.0 REQUIRED)
```

Comment:
I don't think this needs to change? Technically the keyboard can still build against the older framework version. And it wasn't changed for 2.0.0 release, so I think we don't need to change it now, as certainly no new API has been added or used.

Reply:
This is Plugins, not Framework? I noticed it afterwards and I'm a bit confused why it's there at all.

Reply:
I guess we can keep the new version optional. (Some bugs were fixed in the framework so I guess we at least suggest to update that as well).

Reply:
Sure it'll work with 0.99.1, the qmake-based one? Isn't 0.99.2 better?
Btw, doesn't this imply that running it with years old framework versions is a supported use case? With the little resources available, my personal opinion is that upstream should not support that use case and if a downstream wants to mix different version, they should patch the cmake file.

Reply:
I won't expect anyone is likely to be mixing them anyway. However, there's also no need to needlessly bump dependency version requirements. If there was some change in framework which keyboard absolutely depended on to work, but wasn't an API/ABI change, then perhaps it would makes sense. But there's no specific reason to lock the dependency in this way currently.

Reply:
So 0.99.1 will work despite still being based on qmake? Honest question, not a programmer.

Reply:
Yes, the build system is unrelated to the built library.